src/main/java/org/sunbird/user/service/UserUtilityServiceImpl.java

```diff
@@ -969,4 +969,45 @@ private String constructEmailTemplate(String templateName, Map<String, Object> p
 		}
 		return replacedHTML;
 	}
+
+	@Override
+	public Map<String, Object> getUserDetails(String key, String value) {
+		// request body
+		SunbirdApiRequest requestObj = new SunbirdApiRequest();
+		Map<String, Object> reqMap = new HashMap<>();
+		reqMap.put(Constants.FILTERS, new HashMap<String, Object>() {
+			{
+				put(key, value);
+			}
+		});
+		requestObj.setRequest(reqMap);
+
+		HashMap<String, String> headersValue = new HashMap<>();
+		headersValue.put(Constants.CONTENT_TYPE, "application/json");
+		headersValue.put(Constants.AUTHORIZATION, props.getSbApiKey());
```

Comment:
is API key need here ?

Reply:
I take the reference for public boolean isUserExist(String key, String value) and here it is used this way so i used.